x/simulation/account.go

```diff
@@ -32,21 +32,29 @@ func RandomAcc(r *rand.Rand, accs []Account) Account {
 }
 
 // RandomAccounts generates n random accounts
-func RandomAccounts(r *rand.Rand, n int) []Account {
+func RandomAccounts(r *rand.Rand, n int, blackList map[string]bool) []Account {
 	accs := make([]Account, n)
 	for i := 0; i < n; i++ {
-		// don't need that much entropy for simulation
-		privkeySeed := make([]byte, 15)
-		r.Read(privkeySeed)
-		useSecp := r.Int63()%2 == 0
-		if useSecp {
-			accs[i].PrivKey = secp256k1.GenPrivKeySecp256k1(privkeySeed)
-		} else {
-			accs[i].PrivKey = ed25519.GenPrivKeyFromSecret(privkeySeed)
+		for {
```

Comment:
Let’s try to avoid blank for loops if possible. This was an issue in simulation in many places.

Reply:
removed